.github/workflows/env-setup-build-forms-server.yml

```diff
@@ -0,0 +1,43 @@
+name: Env Setup - Build Forms Server
+run-name: Env Setup - Build forms server from ${{ inputs.gitRef }}(form.io tag ${{ inputs.formioTag }})
+
+on:
+  workflow_dispatch:
+    inputs:
+      formioTag:
+        description: "Formio tag, please refer https://github.com/formio/formio/tags to deploy the appropriate tag."
+        type: string
+        required: true
+        default: "v2.5.3"
+      gitRef:
+        description: "Git Ref"
+        required: true
+        default: ""
```

Comment:
why should we need the default as empty?

Reply:
The git ref for build is the reference something like main-65, it does not have any default value.